### PR TITLE
Feature/cleanup

### PR DESCRIPTION
**Pull request by jas88**
- Further tidyup (unnecessary usings, streamlining out vars)

### CODE REVIEW

File: Rdmp.Dicom/PipelineComponents/PrimaryKeyCollisionIsolationMutilation.cs [hunk 1]
```diff
@@ -464,14 +464,8 @@ private string GetPostgreSqlDeleteCommand(TableInfo toDelete,string deleteOnColu
 
                 var usingsStr = string.Join(",",usings.Except(new []{toDelete}).Select(t=>syntax.EnsureWrapped(t.GetRuntimeName(LoadBubble.Raw,_namer))));
 
-                return string.Format("DELETE FROM {0} {1} WHERE {2} {3} = @val",
```
Comment (tznind) [inline Rdmp.Dicom/PipelineComponents/PrimaryKeyCollisionIsolationMutilation.cs:467]: To me the original was more readable,  I think if there are a lot of variables and you want to communicate the core SQL outline being run up front string.Format is ok.

**Review by tznind — CHANGES_REQUESTED** [review]

File: Rdmp.Dicom/LiveVsTemplateComparer.cs [hunk 2]
```diff
@@ -27,7 +27,7 @@ public LiveVsTemplateComparer(ITableInfo table,ImageTableTemplateCollection temp
             // The live table name e.g. CT_StudyTable
             var liveTableName = discoveredTable.GetRuntimeName();
             // Without the prefix e.g. StudyTable
-            var liveTableNameWithoutPrefix = liveTableName.Substring(liveTableName.IndexOf("_")+1);
+            var liveTableNameWithoutPrefix = liveTableName.Substring(liveTableName.IndexOf("_", StringComparison.Ordinal)+1);
```
[the remark this answers is not in the document]
Comment (tznind) [reply Rdmp.Dicom/LiveVsTemplateComparer.cs:30]: Ah ok its only underscore, my mistake.

File: Rdmp.Dicom/Cache/SMICachePathResolver.cs [hunk 3]
```diff
@@ -5,7 +5,7 @@ namespace Rdmp.Dicom.Cache
 {
     public class SMICachePathResolver : ILoadCachePathResolver
     {
-        public string Modality { get; private set; }
+        public readonly string Modality;
```
[the remark this answers is not in the document]
Comment (jas88) [reply Rdmp.Dicom/Cache/SMICachePathResolver.cs]: Reverted to get-only property

File: Rdmp.Dicom/CommandExecution/ExecuteCommandCreateNewImagingDatasetSuite.cs [hunk 4]
```diff
@@ -30,11 +30,11 @@ public class ExecuteCommandCreateNewImagingDatasetSuite : BasicCommandExecution
...
         private readonly DirectoryInfo _projectDirectory;
-        private IExternalDatabaseServer _loggingServer;
-        private IRDMPPlatformRepositoryServiceLocator _repositoryLocator;
-        private ICatalogueRepository _catalogueRepository;
+        private readonly IExternalDatabaseServer _loggingServer;
+        private readonly IRDMPPlatformRepositoryServiceLocator _repositoryLocator;
+        private readonly ICatalogueRepository _catalogueRepository;
 
-        public List<Catalogue> NewCataloguesCreated { get; private set; }
+        public readonly List<Catalogue> NewCataloguesCreated;
```
[the remark this answers is not in the document]
Comment (jas88) [reply Rdmp.Dicom/CommandExecution/ExecuteCommandCreateNewImagingDatasetSuite.cs]: Reverted to get-only property

File: Rdmp.Dicom/PipelineComponents/PrimaryKeyCollisionIsolationMutilation.cs (hunk 1 above)
Comment (jas88) [reply Rdmp.Dicom/PipelineComponents/PrimaryKeyCollisionIsolationMutilation.cs:467]: Changed to be more readable in current PR version

**Review by tznind — APPROVED** [review]

**Pull request merged**